src/invidious/jobs/statistics_refresh_job.cr

```diff
@@ -65,6 +71,7 @@ class Invidious::Jobs::StatisticsRefreshJob < Invidious::Jobs::BaseJob
     }
 
     # Reset playback requests tracker
+    LATCHED_PLAYBACK_STATS["playback"] = STATISTICS["playback"].as(Hash(String, Int64 | Float64))
     STATISTICS["playback"] = {} of String => Int64 | Float64
```

Comment:
Why not just moving this line (the reset) to `load_initial_stats` and then never touch that element again?

Reply:
`load_initial_stats` is only called once and we need to latch the old playback stats each time the statistics is refreshed.

https://github.com/iv-org/invidious/blob/6a88c098f08e54e696b87e3aa478637afdfe4101/src/invidious/jobs/statistics_refresh_job.cr#L41-L49